components/ws-manager-bridge/src/prebuild-updater.ts

```diff
@@ -127,6 +130,7 @@ export class PrebuildUpdater {
                         prebuildID: prebuild.id,
                         status: prebuild.state,
                         workspaceID: instance.workspaceId,
+                        organizationID: workspace?.organizationId,
```

Comment:
Here we can use `info.teamId` instead, and avoid the additional lookup.

Reply:
Addressed in [`bda0663` (#20565)](https://github.com/gitpod-io/gitpod/pull/20565/commits/bda06630679c0ec97edae87e969cf5d9ad453337)